src/Main.java

```diff
@@ -0,0 +1,6 @@
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+    }
+}
```

Comment:
Do you need this file?

Reply:
I don't I guess... I just created the project from the template for the 1st task. I'm gonna delete it.